Makefile

```
@@ -1,7 +1,7 @@
 SHELL := /bin/bash
-
+VERSION := latest
```

i think this is out of scope for this PR but `latest` in is quite a bad practice which can lead to lots of troubles.
can we change it do `?=` to allow us to overdefine the version in the future?
```suggestion
VERSION ?= latest
```

---

Well `latest` it's really kinda of a `dummy` default here since the released manifest will actually have the right version. 

If users happen to run `make deploy` they will also most likely set the IMG or VERSION as well.